### ISSUE TITLE
Allow baseFilter expression type to be "IS NULL" or "IS NOT NULL"

### ISSUE THREAD

**Issue by marks** [timwis/vizwit#152]
Missing from https://github.com/timwis/vizwit/wiki/Base%20Filters

not sure if expression value should be limited to only `NULL`

**Comment by timwis** [comment]
I'm fairly certain this should work by using `"type": "is"` and `"type": "is not"` with `"value": null`. Have you tried it? If you compare the [block you added](https://github.com/marks/vizwit/blob/bf0112b019fcc97fa2c21bdc8d249475742e1f1b/src/scripts/collections/baseprovider.js#L58-L63) to the block below, it's nearly identical (except for "enclose" which shouldn't matter). The reason "in" was explicitly added was because it needed to `.join()` an array and wrap it in parentheses.

**Comment by marks** [comment]
@timwis I am fairly certain the block I added because without it, the query to the provider becomes `&$where=(current_week is not 'NULL')` which is not valid whereas `&$where=(current_week is not NULL)` is.

Let me know if I should proceed with a PR or not

**Comment by timwis** [comment]
@marks Does your config file use `"value": null` or `"value": "null"`? Because the [enclose function](https://github.com/marks/vizwit/blob/bf0112b019fcc97fa2c21bdc8d249475742e1f1b/src/scripts/collections/baseprovider.js#L11-L13) shouldn't put quotes around it if it's not a string. Unless soda-js is doing it...

**Comment by marks** [comment]
@timwis when I do the following without this patch, I get errors. Am I doing something else wrong?

```
    {
      "x": 6,
      "y": 9,
      "width": 6,
      "height": 4,
      "vizwit": {
        "title": "Deaths by MMWR Week",
        "description": "Column chart of the sum of 'current_week' across all diseases",
        "chartType": "bar",
        "provider": "socrata",
        "domain": "healthdata.demo.socrata.com",
        "dataset": "4k9z-fs3i",
        "groupBy": "mmwr_week",
        "aggregateFunction": "sum",
        "aggregateField": "current_week",
        "order": "value desc",
        "baseFilters": [
          {
            "field": "current_week",
            "expression": {
              "type": "is not",
              "value": null
            }
          }
        ]
      }
    },
```

**Comment by marks** [comment]
Merged into master with #162 

Closing this
